core/src/main/java/io/grpc/internal/PickFirstLeafLoadBalancer.java

```diff
@@ -370,7 +432,12 @@ private void shutdownRemaining(SubchannelData activeSubchannelData) {
    */
   @Override
   public void requestConnection() {
-    if (!addressIndex.isValid() || rawConnectivityState == SHUTDOWN) {
+    if (rawConnectivityState == SHUTDOWN) {
+      return;
+    }
+
+    if (!addressIndex.isValid()) {
+      scheduleBackoff();
```

Comment:
Seems this came back. The problem I have with this line is if it is ever needed (it actually schedules the timer), then that means there is a bug somewhere else. Because it means we should have been in backoff but didn't set the timer.